client/components/plans/plan-icon/index.jsx

```diff
@@ -24,12 +24,35 @@ import {
+		const iconPremium = <svg className="plan-icon__premium" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0" y="0" viewBox="0 0 62 61.5"><ellipse className="plan-icon__premium-0" cx="31" cy="30.8" rx="31" ry="30.8"/><path className="plan-icon__premium-1" d="M27.8 46.3v1.9l2.2 5.4c0.2 0.5 0.5 0.7 1 0.7v-7.9L27.8 46.3z"/><path className="plan-icon__premium-2" d="M32 53.6l2.2-5.4v-1.9H31v3.8 4.1C31.5 54.3 31.9 54 32 53.6z"/><path className="plan-icon__premium-3" d="M21.6 18c0.2 2 4.8 28.3 4.8 28.3l4.6 0.1V18H21.6z"/><path className="plan-icon__premium-4" d="M40.5 18H31v23.4 5l4.5-0.1C35.5 46.3 40.1 20.4 40.5 18z"/><path className="plan-icon__premium-5" d="M31 18V0c-3.9 0-7.7 0.7-11.2 2.1 0.7 6 1.8 15.9 1.8 15.9H31z"/><path className="plan-icon__premium-6" d="M31 18h9.5c0 0 1.1-9.9 1.8-15.9C38.8 0.7 35 0 31 0V18z"/></svg>;
+		const iconBusiness = <svg className="plan-icon__business" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0" y="0" viewBox="0 0 124 124"><circle className="plan-icon__business-0" cx="62" cy="62" r="62"/><path className="plan-icon__business-1" d="M57.4 73.8c0-2.5 2-4.6 4.6-4.6V36.8h-9.6v12.1c-8 3.7-13.5 11.7-13.5 21.1 0 4.9 1.5 9.4 4.1 13.1l0 0c15.4 22.4 15.4 22.4 17 24.8h0.3V78.1C58.6 77.4 57.4 75.7 57.4 73.8L57.4 73.8z"/><path className="plan-icon__business-2" d="M85.2 69.8c0-9.4-5.6-17.4-13.5-21.1v-12H62v32.5c2.5 0 4.6 2 4.6 4.6 0 1.9-1.2 3.5-2.9 4.2v29.8H64c1.6-2.4 1.6-2.4 17-24.8l0 0C83.6 79.2 85.2 74.7 85.2 69.8L85.2 69.8z"/><path className="plan-icon__business-3" d="M48.6 32h-2.4c-3.2 0-5.8 2.6-5.8 5.8V38c0 3.2 2.6 5.8 5.8 5.8H62v-4.7V32 0c-7.2 0-14.1 1.2-20.6 3.5 1.1 5.5 2.5 11 4.1 16.5L48.6 32z"/><path className="plan-icon__business-4" d="M62 39.1v4.7h15.8c3.2 0 5.8-2.6 5.8-5.8v-0.2c0-3.2-2.6-5.8-5.8-5.8h-2.4l3.1-12c1.6-5.5 3-11 4.1-16.5C76.1 1.2 69.2 0 62 0v32V39.1z"/></svg>;
 		const { plan, className } = this.props;
 		const planClass = getPlanClass( plan );
 		const classes = classNames( 'plan-icon', planClass, className );
 
+		switch( plan ) {
+			default: svg = <svg className="plan-icon__free" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0" y="0" viewBox="0 0 61.8 61.8"><circle className="plan-icon__free-0" cx="30.9" cy="30.9" r="30.9"/><polygon className="plan-icon__free-1" points="41.4 32.8 30.9 22.5 20.3 32.8 30.9 51.8 "/><path className="plan-icon__free-2" d="M30.9 41.6L30.9 41.6c-2 0-3.7 1.6-3.7 3.6l3.7 6.5 3.7-6.5C34.5 43.2 32.9 41.6 30.9 41.6L30.9 41.6z"/><path className="plan-icon__free-3" d="M25.6 27.7c0 2.9 2.3 5.2 5.2 5.2h0.1c2.9 0 5.2-2.3 5.2-5.2v-0.1V0.4c-1.7-0.3-3.5-0.5-5.3-0.5s-3.6 0.2-5.3 0.5v27.3H25.6z"/><path className="plan-icon__free-4" d="M25.6 27.6V0.4c-1.8 0.3-3.6 0.8-5.3 1.4v31C23.2 32.8 25.6 30.5 25.6 27.6z"/><path className="plan-icon__free-2" d="M41.4 32.8v-31c-1.7-0.6-3.5-1.1-5.3-1.4v27.2C36.1 30.5 38.5 32.8 41.4 32.8z"/></svg>;
+				break;
+			case 'personal-bundle':
+			case 'jetpack_personal':
+			case 'jetpack_personal_monthly': svg = iconPersonal;
```

Comment:
Let's use the constants directly for the case statements eg: `case PLAN_PERSONAL:`.